here-naksha-lib-hub/src/main/java/com/here/naksha/lib/hub/storages/NHSpaceStorageReader.java

```diff
@@ -292,6 +289,11 @@ public void setLockTimeout(long timeout, @NotNull TimeUnit timeUnit) {}
       }
     }
 
+    // Ensure the order of the event handlers is preserved
+    final Space finalSpace = space;
```

Comment:
Any specific reason to not use `space` variable itself (instead of `finalSpace`)?

Reply:
Yes it would be a compiler error: "local variables referenced from a lambda expression must be final or effectively final"

Reply:
Ah, damn Lambda!!